### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by Quintinlouw36** [community/community#12868] [Mobile]
Hi there good day.
Could I please ask for additional details and patterns of how this actually works,it would be a great help as I am very new in the system.

**Comment by ghost** [comment]
GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere. This tutorial teaches you GitHub essentials like repositories, branches, commits, and pull requests.

**Reply by Quintinlouw36** [reply]
Thanks for the feedback!!!
